packages/for-ui/src/badge/Badge.tsx

```diff
@@ -1,25 +1,88 @@
 import React, { ReactNode } from 'react';
-import { BadgeProps } from '@mui/material/Badge';
-import MuiBadge from '@mui/material/Badge';
 
-interface Props extends BadgeProps {
-  className?: string;
-  children: ReactNode;
-}
+import { MdError } from 'react-icons/md';
+import { fsx } from '../system/fsx';
 
-export const Badge: React.FC<Props> = ({ className, badgeContent, children, ...rest }) => {
-  return (
-    <MuiBadge
-      badgeContent={badgeContent}
-      color="primary"
-      componentsProps={{
-        root: {
-          className: `text-shade-dark-default ${className}`,
-        },
-      }}
-      {...rest}
-    >
-      {children}
-    </MuiBadge>
-  );
+type Props = {
+  color?: 'white' | 'gray' | 'primary' | 'native';
+  icon?: ReactNode;
+  type?: 'constant' | 'text' | 'outline';
+  label: string;
+};
+
+export const Badge: React.FC<Props> = ({ color = 'white', icon, type = 'constant', label }) => {
+  switch (type) {
+    case 'constant':
+      return (
+        <div
+          className={fsx([
+            'flex flex-row items-start gap-1 rounded',
+            color === 'white' && 'text-[#0B0D0E]',
+            color === 'primary' && 'text-[#005093]',
+            color === 'native' && 'text-[#C00F50]',
+          ])}
+        >
+          <div className="flex h-6 w-4 flex-row items-start py-1">
+            <MdError className="h-4 w-4" />
```

Comment:
すみません！ここのアイコンも変えられるようにしたく…

```suggestion
            {icon}
```